test_maya.py

```diff
@@ -42,11 +45,30 @@ def test_dt_tz_naive():
     assert d1.hour - d2.hour == 5
 
 
-def test_random_date():
-    d = maya.when('11-17-11')
+def test_random_date(capsys):
```

Comment:
No it's for capturing stdin and stout in pytest, see http://doc.pytest.org/en/latest/capture.html#accessing-captured-output-from-a-test-function